danger-kotlin/src/runnerMain/kotlin/com/danger/runner/cmd/danger-js/DangerJS.kt

```diff
@@ -6,7 +6,7 @@ object DangerJS: DangerJSBridge {
 
     override fun process(command: String, processName: String, args: List<String>) {
         with(Cmd()) {
-            name("$(which danger-$command)")
```

Comment:
You forgot the other parameters.
if I run 
`danger-kotlin pr https://github.com/danger/kotlin/pull/20` 
it should call 
`$(which danger) $command --process $processName https://github.com/danger/kotlin/pull/20`

Reply:
inside the `with(Cmd())` block you can find args as well with the missing stuff

Reply:
Great!, should `$command --process $processName` go on the args then?